signify/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "signify"
+version = "0.4.1"
+authors = ["Jan-Erik Rediger <janerik@fnordig.de>"]
```

Comment:
we can drop this line, alternatively add yourself here

Reply:
Added myself here for now. Happy to drop the line or switch it to "Signify Contributors" if the crate picks up more users and contributors.